ast/builtins.go

```diff
@@ -2615,6 +2618,22 @@ var GraphQLSchemaIsValid = &Builtin{
 	),
 }
 
+/**
+ * Cloud Provider Helper Functions
+ */
+
+var ProvidersAWSSigV4SignReqObj = &Builtin{
+	Name:        "providers.aws.sigv4.sign_req",
```

Comment:
Quick thought: Should we include the signature version in the builtin name? We could just call it `providers.aws.sign_req` and then in the description mention that it supports SigV4. Not sure if `providers.aws.sign_req`  would lead to some confusion if AWS has multiple signing methods.

Reply:
I hadn't spent much thought on that, but I like the idea! :smile: Since the caller provides a config object for the AWS side anyway, we could easily parameterize the signature version in the future, if needed.

Reply:
Rolled the name generalization into my latest changes. :+1: